appender.go

```diff
@@ -195,6 +195,18 @@ func (a *Appender) Flush() error {
 	return nil
 }
 
+func (a *Appender) Clear() error {
+	var errClear error
+	if mapping.AppenderClear(a.appender) == mapping.StateError {
+		errClear = getDuckDBError(mapping.AppenderError(a.appender))
+	}
+
+	if errClear != nil {
+		return getError(invalidatedAppenderError(errClear), nil)
```

Comment:
Maybe we should elaborate more / change this `invalidatedAppenderError` message a bit? If it happens here, we have some type of problem, no? Since we can't clear the appender (which likely contains data causing a timeout or a bug)... Maybe we should try to nudge people to open a bug report?